lightly/active_learning/utils/bounding_box.py

```diff
@@ -0,0 +1,88 @@
+""" Bounding Box Utils """
+
+# Copyright (c) 2020. Lightly AG and its affiliates.
+# All Rights Reserved
+
+
+class BoundingBox:
+    """Class which unifies different bounding box formats.
+
+    Attributes:
+        x0:
+            x0 coordinate (normalized to [0, 1])
+        y0:
+            y0 coordinate (normalized to [0, 1])
+        x1:
+            x1 coordinate (normalized to [0, 1])
+        y1:
+            y1 coordinate (normalized to [0, 1])
+
+    Examples:
+    >>> # simple case
+    >>> bbox = BoundingBox(0.1, 0.2, 0.3, 0.4)
+    >>>
+    >>> # same bounding box in x, y, w, h format
+    >>> bbox = BoundingBox.from_x_y_w_h(0.1, 0.2, 0.2, 0.2)
+    >>>
+    >>> # same bounding box with prior normalization
+    >>> W, H = 100, 100 # get image shape
+    >>> bbox = BoundingBox(10 / W, 20 / H, 30 / W, 40 / H)
```

Comment:
Not sure this example is adding any value

Reply:
I can remove it, thought it might help

Reply:
I would say it does.

Reply:
I would say it does help

Reply:
Then I would add some more explanation. Using words like "prior normalization" is complicated but the scaling of the box to 0-1 is more for a simple thing. So I would use simple language for both. Maybe add more info about the input bounding box dimensions?